src/main/resources/assets/admin/common/js/notify/NotifyManager.ts

```diff
@@ -93,6 +93,12 @@ module api.notify {
             return notificationEl;
         }
 
+        getNotification(messageId: string): NotificationMessage {
+            if (messageId && this.registry[messageId]) {
+                return this.registry[messageId].el;
+            }
+        }
```

Comment:
A non-void method should always return something. ``getNotification`` should return ``null`` if the notification doesn't exist.